domains/trix.json

```
@@ -0,0 +1,13 @@
+{
+    "owner": {
+        "username": "tr1xem",
+        "email": ["kmtechnical2006@gmail.com"]
+    },
+
+    "record": {
+        "TXT": ["_github-pages-challenge-tr1xem.trix=b2b8238a381a3d699cc09cf07a22ae"],
+        "CNAME" : "tr1xem.github.io"
```

Remove TXT since you don't need it for the main domain + you can't use it with CNAME together.
```suggestion
        "CNAME" : "tr1xem.github.io"
```